.devcontainer/post_create.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+sudo apt-get update
+sudo apt-get install -y libaspell-dev
+
+pip install --upgrade \
+    aspell-python-py3 \
+    pip \
+    setuptools \
+    setuptools_scm \
+    wheel
+pip install -e '.[dev]'
```

Comment:
Out of curiosity, why a dev install? So that end-users can modify the dictionary or even the code locally?

Reply:
Yup. That's exactly right. The goal is to give you a single-click development environment for a local copy of Codespell. For your context, your local Codespell folder is automatically mounted to the default directory which is the context where these commands are being run.

I'm looking to mirror [this development setup guide](https://github.com/codespell-project/codespell#development-setup) with the addition of support for all optional tests.